### PR TITLE
octopus:os/bluestore: improve/fix bluefs stats reporting

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/os/bluestore/BlueStore.cc [hunk 1]
```diff
@@ -15880,6 +15884,8 @@ void RocksDBBlueFSVolumeSelector::dump(ostream& sout) {
     sout.setf(std::ios::left, std::ios::adjustfield);
     sout.width(width);
     switch (l + LEVEL_FIRST) {
+    case LEVEL_LOG:
+      sout << "LOG"; break;
     case LEVEL_WAL:
```
Comment (aclamk) [inline src/os/bluestore/BlueStore.cc:15889]: Tip: How about changing in future LEVEL_SLOW to print "BLOCK".

File: src/os/bluestore/BlueStore.cc [hunk 2]
```diff
@@ -15834,14 +15836,15 @@ void RocksDBBlueFSVolumeSelector::dump(ostream& sout) {
     << ", slow_total:" << l_totals[LEVEL_SLOW - LEVEL_FIRST]
     << ", db_avail:" << db_avail4slow << std::endl
     << "Usage matrix:" << std::endl;
-  constexpr std::array<const char*, 7> names{ {
+  constexpr std::array<const char*, 8> names{ {
     "DEV/LEV",
```
Comment (aclamk) [inline src/os/bluestore/BlueStore.cc:15840]: Tip: How about changing "DEV/LEV" to "LEV\DEV"?

**Comment by yuriw** [comment]
[wip-yuri8-testing-2020-07-01-2358-octopus](https://trello.com/c/grbctxQs/951-wip-yuri8-testing-2020-07-01-2358-octopus)

**Comment by yuriw** [comment]
@ifed01 this passed tests and approved
pls resolve conflict and merge at will

**Comment by ifed01** [comment]
jenkins make test check

**Comment by ifed01** [comment]
jenkins test make check

**Pull request merged**